dockerfiles/alpine_3.5_2.2

```diff
@@ -96,8 +96,26 @@ RUN set -x \
         ./configure; \
         make -j ; \
         cp .libs/libprofiler.so* /usr/local/lib;) \
-    && (GOPATH=/usr/src/go go get github.com/google/pprof; \
-        cp /usr/src/go/bin/pprof /usr/local/bin) \
+    && : "---------- pprof for gperftools ----------" \
+    && : "To avoid of the issue:" \
+    && : "'The master branch is Incompatible with go version < 1.13 #538'" \
+    && : "https://github.com/google/pprof/issues/538" \
+    && : "use latest workable commits with the old GO versions" \
+    && ( export GOPATH=/root/go && \
+       export PATH=${GOPATH}/bin:/usr/local/go/bin:$PATH && \
+       export GOBIN=$GOROOT/bin && \
+       mkdir -p ${GOPATH}/src ${GOPATH}/bin && \
+       export GO111MODULE=on && \
```

Comment:
Apline 3.5 offers go-1.7.3-r0, so GO111MODULE variable does not change anything here.